app/components/report-wrong-data.js

```diff
@@ -54,6 +54,6 @@ export default class ReportWrongDataComponent extends Component {
       this.association ? this.association.name : ''
     } %0D%0A vCode: ${
       this.vCode
-    } %0D%0A Aan te passen op het ${url} (Verenigingsloket), pagina 'Mijn gegevens'.`;
+    } %0D%0A Aan te passen op het ${url} (Verenigingsregister), pagina 'Mijn gegevens'.`;
```

Comment:
Not sure if this one should be renamed, since the url is `verenigingsloket.be` but it seemed better to be consistent?

Reply:
I'll roll this back. We can do it in a follow-up PR once we get confirmation.